packages/i18n/src/locales/en/users.json

```
@@ -1,10 +1,10 @@
 {
   "table": {
-    "title": "users table",
+    "title": "Users table",
```

Lets remove the "table" in title.
```suggestion
    "title": "Users",
```